massa-client/src/main.rs

```diff
@@ -51,13 +52,33 @@ struct Args {
     /// Enable a mode where input/output are serialized as JSON
     #[structopt(short = "j", long = "json")]
     json: bool,
+    #[structopt(short = "p", long = "pwd")]
+    /// Wallet password
+    password: Option<String>,
 }
 
 #[derive(Serialize)]
 struct JsonError {
     error: String,
 }
 
+fn ask_password(wallet_path: &PathBuf) -> String {
+    if wallet_path.is_file() {
+        Password::new()
+            .with_prompt("Enter wallet password")
+            .interact()
+            .expect("Input error")
+    } else {
+        let pwd = Password::new()
+            .with_prompt("Enter new password for wallet")
+            .with_confirmation("Confirm password", "Passwords mismatching")
+            .interact()
+            .expect("Input error");
```

Comment:
* the panic message is not explicit enough
* maybe just reask for the password instead of panicking